sdcm/cluster_gce.py

```diff
@@ -11,14 +11,22 @@ class CreateGCENodeError(Exception):
     pass
 
 
-def gce_create_metadata(extra_meta=None):
+def gce_create_metadata_old(extra_meta=None):
     tags = cluster.create_common_tags()
     tags['startup-script'] = cluster.Setup.get_startup_script()
     if extra_meta:
         tags.update(extra_meta)
     return tags
 
 
+def gce_create_metadata(extra_meta=None):
```

Comment:
why did you create this "new" method?
Without `startup-script` nothing will work

Reply:
I've removed old one, it turned out that it did not work for metadata manipulation